server/clients/jsonApiClient.js

```diff
@@ -0,0 +1,35 @@
+const axios = require('axios');
+const httpAdapter = require('axios/lib/adapters/http');
+const { logger } = require('../utils/logger');
+
+class JsonApiClient {
+  constructor(baseURL) {
+    this.client = axios.create({
+      baseURL,
+      timeout: 30000,
```

Comment:
hardcoded to a relatively large timeout - before there was no timeout and calls could potentially block forever.
We could look at extracting this out into config..